### ISSUE TITLE
Skip duplicate lines

### ISSUE THREAD

**Issue by twe4ked** [sharkdp/hexyl#79]
`hexdump` will skip duplicate lines and output a single `*` for any number of duplicate lines. Would you accept a PR that implements that behaviour?

```
0000280 04 00 00 00 2a 00 00 00 00 00 00 00 00 00 00 00
0000290 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
*
0000300 00 00 00 00 00 00 00 00 ac 0f 00 00 01 00 00 00
0000310 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
*
0000330 0c 00 00 00 38 00 00 00 18 00 00 00 02 00 00 00
0000340 01 fa e4 04 00 00 01 00 2f 75 73 72 2f 6c 69 62
```

**Comment by awidegreen** [comment]
That feature was merged with #59 or are you thinking about something else?

**Comment by twe4ked** [comment]
Whoops! I'm on version 0.3.0, that was the problem. Thanks :)
